.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,19 @@
-## Related issue
-One line for each issue. If merging this pull request should auto-close the issue on that line,
-preface the issue URL/number with "Fixes". Every pull request should have at least one issue
+**Link the Issue(s) this Pull Request is related to.**
+One line for each Issue. If merging this Pull Request should auto-close the Issue on that line,
```

Comment:
@bcipriano, I have done Issue and all capitalized but @sharifsalah has commented to do it in lowercase.
I think, Now it looks much better.

Reply:
Thanks for pointing that out! I fixed the capitalization.

Regarding the checklist -- I thought about it some more and I think it's redundant. The headers of each section are a kind of checklist already, since we're asking the PR author to fill in the information for each. So I don't think it's necessary to ask them to fill in that info AND update the checklist markdown.